kubernetes/README.md

```diff
@@ -0,0 +1,86 @@
+# Deploying OpenWhisk on Kubernetes
+
+[![Build Status](https://travis-ci.org/openwhisk/openwhisk-devtools.svg?branch=master)](https://travis-ci.org/openwhisk/openwhisk-devtools)
+
+The collection of scripts and Kubernets pods can be used to
+run OpenWhisk in a Kubernets cluster.
+
+#### Kubernetes Requirments
+
+* Kubernetes needs to be version 1.5+
+* Kubernetes pods can receive public addresses
+* Kubernetes has DNS deployed
```

Comment:
Can you provide more details here?
Do I need to configure a file in kubernetes?
Do I need a DNS name configure in my etc/hosts file?